app/views/appliance_calculator/daily_usage_creation/steps/completed.html.haml

```diff
@@ -1,8 +1,16 @@
 = render ApplianceCalculator::LastAddedApplianceComponent.new(flash.notice)
 
-%form{ action: appliance_calculator_path, method: "GET"}
-  = render CitizensAdviceComponents::Button.new(type: :submit) do
-    Add another appliance
-
 = render ApplianceCalculator::MostExpensiveApplianceComponent.new(usages)
-= render ApplianceCalculator::UsagesTableComponent.new(usages: usages, unit_rate: saved_unit_rate)
\ No newline at end of file
+= render ApplianceCalculator::UsagesTableComponent.new(usages: usages, unit_rate: saved_unit_rate)
+
+%p
+  If you want to compare appliances using a different rate, press ‘Clear list’ to restart the calculator.
```

Comment:
Are the marks around `Clear list` supposed to be speech marks? They look slightly odd, which I know can happen sometimes if you're copying text